pkg/yurthub/proxy/local/local.go

```diff
@@ -87,6 +96,21 @@ func (lp *LocalProxy) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 	}
 }
 
+// localServerVersion handles /version GET requests when remote servers are unhealthy
+func (lp *LocalProxy) localServerVersion(w http.ResponseWriter, req *http.Request) error {
+
+	versionInfo, err := lp.cacheMgr.QueryBytes(req)
+	copyHeader(w.Header(), req.Header)
+	w.WriteHeader(http.StatusOK)
+	nw, err := w.Write(versionInfo)
+	if err != nil {
+		klog.Errorf("write resp for /version request when cluster is unhealthy, expect %d bytes but write %d bytes with error, %v", nw, err)
+	}
+	klog.V(5).Infof(" get /version info when cluster is unhealthy")
```

Comment:
if err occurs and we failed to get `/version` from storage, this log should not output.